src/keyboard/textinput_test.js

```diff
@@ -753,6 +753,18 @@ module.exports = {
         assert.equal(editor.getValue(), "");
         sendEvent("input", {key: {inputType: "historyRedo"}});
         assert.equal(editor.getValue(), "x");
+    },
+
+    "test: text input aria label": function() {
```

Comment:
Can we also add same test but without `textInputAriaLabel ` being set.

Reply:
good point, added